src/monitor-api-requests.ts

```diff
@@ -0,0 +1,158 @@
+import { MonitorParams } from "monitors";
+import { Response } from "node-fetch";
+import * as Serverless from "serverless";
+import fetch from "node-fetch";
+
+export class InvalidAuthenticationError extends Error {
+  constructor(message: string) {
+    super(...message);
+    this.name = "Invalid Authentication Error";
+    this.message = message;
+  }
+}
+
+interface QueriedMonitor {
+  query: string;
+  id: number;
+  name: string;
+  tags: string[];
+}
+
+export async function createMonitor(
+  serverlessMonitorId: string,
+  monitorParams: MonitorParams,
+  monitorsApiKey: string,
+  monitorsAppKey: string,
+) {
+  const response: Response = await fetch("https://api.datadoghq.com/api/v1/monitor", {
+    method: "POST",
+    headers: {
+      "DD-API-KEY": monitorsApiKey,
+      "DD-APPLICATION-KEY": monitorsAppKey,
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(monitorParams),
+  });
+
+  if (response.status === 200) {
+    return true;
+  } else if (response.status === 403) {
+    throw new InvalidAuthenticationError("Could not perform request due to invalid authentication");
+  } else if (response.status === 400) {
+    console.log(`Invalid Syntax Error: Could not perform request due to incorrect syntax for ${serverlessMonitorId}`);
```

Comment:
If we fail to create a monitor due to invalid syntax, do we want to make the deploy fail?